src/components/CountdownTimer/components/TimerList/TimerList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { TimerItem } from './components/TimerItem';
+import './TimerList.scss';
+
+interface TimerListProps {
+  values: number[];
+}
+export const TimerList = ({ values }: TimerListProps) => {
+  const timerItems = ['days', 'hours', 'minutes', 'seconds'].map((el, i) => {
+    return { value: values[i], caption: el };
```

Comment:
1. rename el and i

Reply:
refactored: added enum, deleted one map, changed variables' names